### PR TITLE
Implementing inspect on the wrapper

### PR DESCRIPTION
**Pull request by sunesimonsen**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -15,12 +15,107 @@ const unexpectedEnzyme = {
...
-        const inspected = reactWrapper.exists()
-          ? reactWrapper.getElement()
-          : reactWrapper.root().html();
+      inspect: function(reactWrapper, depth, output, inspect) {
+        if (!reactWrapper.exists()) {
+          return output.jsKeyword('null');
+        }
+
+        if (reactWrapper.length > 1) {
+          return output.appendItems(reactWrapper.map(node => node), '\n');
```
Comment (albertfdp) [inline src/index.js:24]: Not sure if there is any benefit on using [#getNodes](https://github.com/airbnb/enzyme/blob/master/docs/api/ReactWrapper/getNodes.md)

File: src/index.js [hunk 2]
```diff
@@ -15,12 +15,107 @@ const unexpectedEnzyme = {
...
+        }
+
+        const startTag = output.clone();
+        startTag.text('<');
+        startTag.jsKeyword(reactWrapper.name());
+
+        const props = reactWrapper.props();
+        Object.keys(props).forEach(key => {
+          if (key === 'children') {
```
Comment (albertfdp) [inline src/index.js:33]: Would it make sense to check for `children` to also be of type `string|node`? Maybe we want to do something different for render props?

**Review by albertfdp — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (sunesimonsen) [reply src/index.js:24]: I think the above will actually map to new React Wrappers which I need.

File: src/index.js (hunk 2 above)
Comment (sunesimonsen) [reply src/index.js:33]: Don't we always want to render the childrens inside the tag, this is just preventing us from inspecting them as a prop.

Comment (sunesimonsen) [reply src/index.js:33]: Let's see what happens, it is a little bit hard to predict all problems that can arise. We need some more tests and real world usage.

**Pull request merged**